task/01-strings-tasks.js

```diff
@@ -201,7 +219,49 @@ function extractEmails(str) {
  *
  */
 function getRectangleString(width, height) {
-    throw new Error('Not implemented');
+    const symbols = ['┌', '┐', '└', '┘', '─', '│']
+
+    let res = ''
+
+    for (let i = 0; i < height; i++) {
+
+        for (let j = 0; j < width; j++) {
+            if (i === 0 && j === 0) {
+                res += symbols[0]
+                continue
+            }
+
+            if (i === 0 && j === width - 1) {
+                res += symbols[1]
+                continue
+            }
+
+            if (i === height - 1 && j === width - 1) {
+                res += symbols[3]
+                continue
+            }
+
+            if (i === height - 1 && j === 0) {
+                res += symbols[2]
+                continue
+            }
+            
+            if ((i > 0 && i < height) && (j === 0 || j === width - 1)) {
+                res += symbols[5]
+                continue
+            }
+            
+
+            if (i === 0 || i === height - 1) {
+                res += symbols[4]
+                continue
+            }
+
+            res += ' '
+        }
+        res += '\n'
+    }
+    return res
```

Comment:
It is great that you have come up with a solution for this task, but you can try to find more readable and cleaner one

Hint: use `repeat` method